pkg/reconciler/common/install_test.go

```diff
@@ -140,10 +140,9 @@ func TestUninstall(t *testing.T) {
 	crd := clusterScopedResource("apiextensions.k8s.io/v1beta1", "CustomResourceDefinition", "test-crd")
 
 	// Deliberately mixing the order in the manifest.
-	in := []unstructured.Unstructured{deployment, role, roleBinding, clusterRole, clusterRoleBinding, crd}
+	in := []unstructured.Unstructured{crd, deployment, role, roleBinding, clusterRole, clusterRoleBinding}
```

Comment:
why is this change required?

Reply:
he deletion happens back to front, so the ordered will be reveresed.
earlier the pass condition is that crds will not be deleted. then the order where we put the crd in the first list did not matter, as crd will not be in the deletes list